definitions/npm/supports-webp_v1.x.x/flow_v0.84.x-/supports-webp_v1.x.x.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+declare module "supports-webp" {
+  declare module.exports: boolean;
```

Comment:
https://github.com/fregante/supports-webp/blob/v1.0.6/index.js

Reply:
Please prefer ES modules. Especially because the package is using it.

Reply:
```suggestion
  declare export default boolean;
```